lib/utils/config/src/lib/buffer-config.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+const oauth = require('abacus-oauth');
+const vcapenv = require('abacus-vcapenv');
+
+module.exports.bufferConfig = (getFromEnv) => ({
+  token: (authServerUri) => { 
```

Comment:
Should this logic be here? It does not seem like "configuration" - this call (.cache()) actually results in some rest requests to uaa.